packages/backend-function/API.md

```diff
@@ -93,6 +93,7 @@ export type FunctionProps = {
     entry?: string;
     timeoutSeconds?: number;
     memoryMB?: number;
+    ephemeralStorageSize?: number;

```

Comment:
Can we suffix this with `MB` ? we do that in property above.

Reply:
Some context:
It seems that CDK is just passing through this value:
see https://github.com/aws/aws-cdk/blob/b5c21890cf67ddda78c18eb0425f11a59fcadb34/packages/aws-cdk-lib/aws-lambda/lib/function.ts#L1057
and perhaps multiplies it by 1024 if necessary.
see https://github.com/aws/aws-cdk/blob/62a0638e325024fc0469ed50f9accd448d87bac7/packages/aws-cdk-lib/core/lib/size.ts#L133

But the lambda claims it's `MB` not `MiB` here https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-lambda-function-ephemeralstorage.html .

To confirm this. It might be good to deploy lambda with `ephemeralStorageSize` given to CDK construct and checking what value is printed in the AWS Console.

Reply:
@sobolk
In CloudFormation and AWS Console, the unit of ehpemeralStorageSize for Lambda was `MB`.
I understood that only CDK uses `MiB` as a unit and it is appropriate to give MB to the suffix in Amplify. 
I fixed the part of JSDoc for ephemeralStorageSizeMB that used `MiB` has been corrected to `MB`. 
Thank you very much.